### ISSUE TITLE
Base de données dans un autre répertoire que le code

### ISSUE THREAD

**Issue by GwilhermPoullennec** [inventaire-des-orgues/portail#96]
Par exemple (?) :

- /data/db_saved/db
- /data/db_saved/img
- /data/db/db
- /data/db/img

**Comment by LucasBerbesson** [comment]
Tu veux faire une sauvegarde ?

**Comment by GwilhermPoullennec** [comment]
Oui, cela me semblait sain en vu de sauvegardes de scinder le répertoire code du répertoire données (BD sqlite et images, qui vont être assez nombreuses).
Pour les modifications faites hors ligne avec les codes infects stockés dans le dépôt Paillasse, je pensais envoyer les données récupérées depuis internet dans le répertoire /data/ via Git sans risque d'interférer avec le code.

**Comment by LucasBerbesson** [comment]
Backup de la base de données toutes les nuits à 1h ici : /home/fabdev/data/db_saved
10 jours d'historique gardés
